File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/database/model/RealmSavedAddress.java
package com.lattis.ellipse.data.database.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.Required;

public class RealmSavedAddress extends RealmObject {

    @Required
    @PrimaryKey
    private String placeId;
    private String name;
    private Double latitude;
    private Double longitude;


    public String getId() {
        return placeId;
    }
    public void setId(String id) {
        this.placeId = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Double getLatitude() {
        return latitude;
    }

    public void setLatitude(Double latitude) {
        this.latitude = latitude;
    }

    public Double getLongitude() {
        return longitude;
    }

    public void setLongitude(Double longitude) {
        this.longitude = longitude;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/database/model/RealmSavedAddress.kt
package com.lattis.data.database.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.Required

open class RealmSavedAddress : RealmObject() {
    @Required
    @PrimaryKey
    var id: String? = null
    var address1: String? = null
    var address2: String? = null
    var latitude: Double? = null
    var longitude: Double? = null

}